app/services/clever_integration/creators/students.rb

```diff
@@ -41,6 +41,6 @@ def self.create_student(parsed_student_response)
         role: 'student'
       })
     end
-    student.reload
+    student.reload if student.id?
```

Comment:
Why are we even reloading here? Can we just nix this? This file feels super enigmatic to me.